tools/nodejs_api/src_cpp/node_util.cpp

```diff
@@ -137,6 +137,16 @@ Napi::Value Util::ConvertToNapiObject(const Value& value, Napi::Env env) {
     case LogicalTypeID::INTERNAL_ID: {
         return ConvertNodeIdToNapiObject(value.getValue<nodeID_t>(), env);
     }
+    case LogicalTypeID::MAP: {
+        Napi::Object napiObj = Napi::Object::New(env);
+        for (auto i = 0u; i < NestedVal::getChildrenSize(&value); ++i) {
+            auto childVal = NestedVal::getChildVal(&value, i);
+            auto key = NestedVal::getChildVal(childVal, 0)->toString();
```

Comment:
Should we always convert key to string?

Reply:
Since NAPI does not support Javascript `Map` type for now, we are just outputting plain Javascript `Object` for now (which is [commonly used by Javascript developers as a dictionary](https://stackoverflow.com/questions/18541940/map-vs-object-in-javascript)). The `Object` type only supports strings as keys.